docs/source/backends/wireguard.rst

```diff
@@ -103,6 +103,20 @@ key name        type    default   allowed values
                                   underscores
 ``port``        integer ``51820`` integers
 ``private_key`` string            base64-encoded private key
+``dns``         list    ``[]``    list of strings containing DNS servers
+``mtu``         integer ``1280``  MTU set on the interface
```

Comment:
I would change the description here to just "interface MTU".

I was confused by the default value here because I remember the default MTU being 1420, but on IPv6 is 1280.

I just realized this is not a great situation. I am thinking about the easiest acceptable solution here.. 

We could set the default to `1420` both here and in the schema, and add a help text (`description` property in the schema) which says: "Interface MTU, set to 1280 if using IPv6."

Alternatively, we could remove the default and state in the description to use 1420 or 1280 whether using ipv4 or ipv6, however, I using IPv4 is the norm, so the first solution is probably going to be the most comfortable for most users.

What do you think?

Reply:
I like the first solution more. I have made changes accordingly.